docs/mathematics/matrix-addition.mdx

```diff
@@ -0,0 +1,183 @@
+---
+title: Matrix Addition
+slug: /mathematics/matrix-addition
+---
+## Introduction
+
+In mathematics, matrix addition is the operation of adding two matrices by adding the corresponding entries together. Two matrices must have an equal number of rows and columns to be added.
+
+## Explanation
+
+Consider two matrices `A` and `B`, the sum of two matrices `A` and `B` will be a matrix that has the same number of rows and columns as `A` and `B`. The sum of `A` and `B` denoted `A` + `B`, is computed by adding corresponding elements of `A` and `B`
+ 
+For Example 
+
+![image](https://i.postimg.cc/Jn8yyrzg/image.png)
+
+## Algorithm
+
```

Comment:
The algorithm can be much more plain text-based, please make changes accordingly.